resources/styles/components/course-listings.less

```diff
@@ -49,18 +49,14 @@
     }
 
     &[data-category=physics] {
-      .tutor-course-item {
-        .tutor-physics-cover-background();
-      }
+      &[data-title=Physics]{ .tutor-course-item { .tutor-physics-cover-background(); } }
```

Comment:
Is there an enumeration of all the values for `catalog_offering_identifier`? Seems odd that we'd continue to special-case based on the title of a course